builders/build/local/adding-built-in-pallet.md

```diff
@@ -41,13 +50,16 @@ Therefore, the first step, is to declare the dependency and make it available to
 [dependencies]
 ...
 pallet-assets = { 
-   git = "https://github.com/paritytech/polkadot-sdk", 
-   branch = "master", 
+   git = "https://github.com/moondance-labs/substrate", 
+   branch = "tanssi-polkadot-v0.9.43", 
    default-features = false 
 }
 ...
 ```
 
+!!! note
+    Our engineering team actively contributes to the Substrate development by fixing issues and enhancing functionalities. As a result, the Tanssi fork repository stays frequently ahead of the official one. That is why this example references a built-in module from a Tanssi repository instead of the official one.
```

Comment:
```suggestion
    Our engineering team actively contributes to the Substrate development by fixing issues and enhancing functionalities. As a result, the Tanssi fork repository frequently stays ahead of the official one. That is why this example references a built-in module from a Tanssi repository instead of the official one.
```